src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/AbstractTask.groovy

```diff
@@ -70,4 +70,22 @@ abstract class AbstractTask extends DefaultTask {
         return (project.liberty.userDir == null) ? new File(installDir, 'usr') : new File(project.liberty.userDir)
     }
 
+    protected List<String> buildCommand (String operation) {
+        List<String> command = new ArrayList<String>()
+        boolean isWindows = System.properties['os.name'].toLowerCase().indexOf("windows") >= 0
+        String installDir = getInstallDir(project).toString()
+
+        if (isWindows) {
+            // println ("Windows detected") 
```

Comment:
take these two println commands out as well.  The if makes it obvious what it is windows.